verification/curator-service/ui/src/components/NewCaseForm.tsx

```diff
@@ -377,7 +383,9 @@ class NewCaseForm extends React.Component<Props, NewCaseFormState> {
                         </nav>
                         <div className={classes.form}>
                             <Form>
-                                <Demographics></Demographics>
+                                <Demographics
+                                    setFieldValue={setFieldValue}
```

Comment:
so you didn't need `setTouched` as well to do: `onBlur={ () => setTouched({ [name]: true }) }` ?

Reply:
I suppose we could, but we don't use the value of touched anywhere so I'm not sure if its worth adding.

Reply:
ok fine, the docs seem to imply this will help validating the field (otherwise perhaps only the initial value is checked), and this is done for all other fields automatically because we're using <Field> which sets onBlur and onChange automatically.
https://jaredpalmer.com/formik/docs/api/formik#settouched-fields--field-string-boolean--shouldvalidate-boolean--void

But if things work as is let's keep it simple then :)

Reply:
Sorry I deleted my comment after I realized it was wrong but before I saw your second comment :P

I added it in for just the reason you said - if there's an error on the field it shouldn't be shown until touched is set to true. Added it in.